search-app/src/cmr/search/data/complex_to_simple_converters/spatial.clj

```diff
@@ -189,6 +190,18 @@
 
              (keys crossings-map))))))
 
+(defn- extract-operator
```

Comment:
I think it would be good to return validation error if the `operator` in `options[spatial][operator]` is not `exclude_br` or `every`.

Reply:
Currently if you try an invalid operator in options such as `options[spatial][abc]` then the API will return 
`{
"errors": [
"Option [asd] is not supported for param [spatial]"
]
}`

It will return this error for any spatial parameter not defined in `(def spatial-options #{:ignore-br :every})` in file `search.services.parameters.parameter-validation.clj`

Is this sufficient or should I implement an additional validation?

Reply:
This is enough. Maybe I missed it, but I don't see a validation test.

Reply:
Added validation test case for invalid spatial options.